pkg/debug/checkups/server_version.go

```diff
@@ -0,0 +1,68 @@
+package checkups
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/kolide/launcher/pkg/agent/types"
+)
+
+type (
+	serverVersionCheckup struct {
+		k       types.Knapsack
+		status  Status
+		summary string
+		data    map[string]any
+	}
+)
+
+func (svc *serverVersionCheckup) Data() map[string]any  { return svc.data }
+func (svc *serverVersionCheckup) ExtraFileName() string { return "" }
+func (svc *serverVersionCheckup) Name() string          { return "Server Version" }
+func (svc *serverVersionCheckup) Status() Status        { return svc.status }
+func (svc *serverVersionCheckup) Summary() string       { return svc.summary }
+
+func (svc *serverVersionCheckup) Run(ctx context.Context, extraFH io.Writer) error {
+	svc.data = make(map[string]any)
+	if !svc.k.KolideHosted() {
+		return nil
+	}
+
+	httpClient := &http.Client{Timeout: requestTimeout}
+	serverEndpoint := fmt.Sprintf("%s/version", svc.k.KolideServerURL())
```

Comment:
Is this for helping us debug k2 <=> launcher issues, so we know what version of k2 was running when we saw an issue, or for something else?

Reply:
yes i believe this is the original intent because we separately have resolution and connectivity checks for this host- but auditing this before answering I'm realizing this info will already be included in the connectivity check (which looks at `/version` by default), so I think we can pull this out cleanly. good question